src/test/librgw_file_nfsns.cc

```diff
@@ -221,7 +221,9 @@ TEST(LibRGW, SETUP_HIER1)
     if (do_create) {
       /* create objects directly */
       std::vector<std::string> obj_names =
-	{"foo/bar/baz/quux",
```

Comment:
I think this is incorrect;  one of the behaviors we need to test is the ability of librgw_file to expand path segments including "virtual" directories;  this change removes an intentional implied path sequence by creating all the segments